app/app.go

```diff
@@ -97,7 +97,7 @@ import (
 )
 
 const (
-	AccountAddressPrefix = "celes"
+	AccountAddressPrefix = "celestia"
 	Name                 = "celestia-app"
 	// BondDenom defines the native staking token denomination.
 	BondDenom = "uceles"
```

Comment:
Should this still be uceles?

Reply:
This PR is only about chaning the account prefix. Not the token name. 
There `uceles` and `celes` should be `utia` and `tia`. See: https://github.com/celestiaorg/celestia-app/issues/279

Reply:
Got it.